Composer/packages/client/src/components/AddRemoteSkillModal/CreateSkillModal.tsx

```diff
@@ -63,7 +63,12 @@ export const getSkillManifest = async (projectId: string, manifestUrl: string, s
     });
     setSkillManifest(data);
   } catch (error) {
-    setFormDataErrors({ ...error, manifestUrl: formatMessage('Manifest URL can not be accessed') });
+    const httpMessage = error?.response?.data?.message;
+    const message = httpMessage?.match('Unexpected string in JSON')
+      ? formatMessage("Error attempting to parse Skill manifest. There could be an error in it's format.")
+      : formatMessage('Manifest URL can not be accessed');
```

Comment:
Hi @cwhitten , I made some update base on your change here . Is this your intention?

Reply:
Yes thank you @zhixzhan !